src/zip/Zip.js

```diff
@@ -136,6 +136,16 @@ class Zip {
         }
     }
 
+    removeFile = (fileName) => {
+        try {
+            const file = this.fileRecord[this.fileRecord.length - 1]
+            Utils.log(`Remove file: ${file.name}`)
+            this.fileRecord = this.fileRecord.filter(record => record.name !== fileName)

```

Comment:
It seems strange that we first get the last file and saying that we're removing it when we actually filter by `fileName` argument. Also, doesn't seem like we need `try/catch` here... should it be just like this?
```js
removeFile = (fileName) => {
  Utils.log(`Remove file: ${fileName}`)
  this.fileRecord = this.fileRecord.filter(record => record.name !== fileName)
}
```

Reply:
Or perhaps if we assume there is just 1 file then it's better to do like this?
```js
Utils.log(`Remove file: ${fileName}`)
const fileIndex = this.fileRecord.findIndex(record => record.name === fileName)
if (fileIndex !== -1) this.fileRecord.splice(fileIndex, 1)
```
Not a big deal though

Reply:
Heh, good call. This was a bad copy paste on my part.